code/game/objects/items/stacks/medical_packs.dm

```diff
@@ -77,6 +77,9 @@
 							"<span class='green'>You apply [src] on [M].</span>")
 		use(1)
 
+/obj/item/stack/medical/attack_self(mob/user)
+	attack(user, user)
```

Comment:
Please move the code out of attack, making a new proc with a new name, and call that instead
```suggestion
	start_apply(user, user)
```